packages/common/src/graphql/schema/directives.ts

```diff
@@ -6,4 +6,5 @@ import gql from 'graphql-tag';
 export const directives = gql`
   directive @derivedFrom(field: String!) on FIELD_DEFINITION
   directive @entity on OBJECT
+  directive @index(unique: Boolean) on FIELD_DEFINITION
```

Comment:
I saw you have index  type enum, so the directive should have another optional property to set index type

Reply:
Yes, I defined the enum but not plan to implement it in this PR

Reply:
Probably don't have to let user to decide index type, so we can enforce the best practice